addons/addon-base/packages/services/lib/__test__/s3-service.test.js

```diff
@@ -224,7 +224,7 @@ describe('S3Service', () => {
       );
     });
 
-    it('should successfully list all object', async () => {
+    it('should successfully list all object without Truncated', async () => {
```

Comment:
Can we add assertions to this test and all the other tests where we missed assertions due to faulty mocking.